catalog-rest-service/src/main/resources/json/schema/entity/feed/thread.json

```diff
@@ -48,6 +48,23 @@
       "description": "User or team this thread is addressed to in format <#E/{entities}/{entityName}/{field}/{fieldValue}.",
       "$ref": "../../type/basic.json#/definitions/entityLink"
     },
+    "createdBy": {
+      "description": "User who created the thread.",
+      "type": "string"
+    },
```

Comment:
We can go with this for now. But perhaps we should take this information in the first post in the future.